### ISSUE TITLE
Idea plugin can't find methods inside the facade.

### ISSUE THREAD

**Issue by anburocky3** [laravel-idea/plugin#193]
Take this as example:

I'm using this:
```php
if ($authUser = User::where('github_id', $githubUser->id)->first()) {
     return $authUser;
}
```
I have imported the class(`use App\Models\User;`) at the top

But still i have this problem like, 
[![Screenshot from 2021-01-25 05-28-09](https://user-images.githubusercontent.com/15264938/105647823-30cdce00-5ece-11eb-9f69-d4262071f263.png)](url)

I can see that, it can be fixed with ide_helper, (https://stackoverflow.com/questions/29439753/eloquent-orm-code-hinting-in-phpstorm), but it has to work with Laravel Idea right?

**Comment by adelf** [comment]
Yes. Please read here - https://laravel-idea.com/docs/eloquent

**Comment by anburocky3** [comment]
Thanks. How about the custom facade, I'm using this package (https://github.com/alaouy/Youtube)
But whenever i call the `Youtube::getChannelByName()` it shows method can't be found inside the `Method 'getChannelByName' not found in Alaouy\Youtube\Facades\Youtube`
![Screenshot from 2021-01-25 05-34-56](https://user-images.githubusercontent.com/15264938/105647979-19dbab80-5ecf-11eb-8d23-2cf570e4ff55.png)

**Comment by adelf** [comment]
Facade helper code generation isn't implemented yet.

**Comment by anburocky3** [comment]
Is it on process stage? If so, what may be the ETA?

**Comment by adelf** [comment]
Not planned yet.

**Comment by adelf** [comment]
Facade Helper Generator is implemented in the 4.1.1 version.
